sdk/documentintelligence/azure-ai-documentintelligence/samples/aio/sample_analyze_addon_barcodes_async.py

```diff
@@ -58,7 +58,6 @@ async def analyze_barcodes():
             "sample_forms/add_ons/barcodes.jpg",
         )
     )
-    # [START analyze_barcodes]
```

Comment:
Are we not using these snippets anywhere?

Reply:
Yes. I don't see we are exposing async code snippet in README.